docker/chronos-nightly/README.md

```diff
@@ -13,14 +13,49 @@ cp docker/chronos-nightly/Dockerfile ./Dockerfile
 ```
 Then build your docker image with Dockerfile:
 ```bash
-sudo docker build -t chronos-nightly:b1 . # You may choose any NAME:TAG you want.
+sudo docker build \
+    --build-arg model=pytorch \
+    --build-arg auto_tuning=y \
+    --build-arg hardware=single \
+    --build-arg extra_dep=n \
+     -t chronos-nightly:b1 . # You may choose any NAME:TAG you want.
+```
+You may specify some build args to install chronos with necessary dependencies according to your own needs.
+The build args are similar to the install options in [Chronos documentation](https://bigdl.readthedocs.io/en/latest/doc/Chronos/Overview/chronos.html#install).
+
+```
+model: which model or framework you want. 
+       value: pytorch
+              tensorflow
+              prophet
+              arima
+              ml (for machine learning models).
+
+auto_tuning: whether to enable auto tuning.
+             value: y (for yes)
+                    n (for no).
+
+hardware: run chronos on a single machine or a cluster.
+          value: single
+                 cluster
+
+extra_dep: whether to install some extra dependencies like onnx and jupyter.
+           value: y (for yes)
+                  n (for no)
+           if specified to y, the following dependencies will be installed:
+           neural_compressor, onnxruntime, onnx, tsfresh, 
+           prometheus_pandas, xgboost, jupyter
 ```
 (Optional) Or build with a proxy:
 ```bash
 # typically, you need a proxy for building since there will be some downloading.
 sudo docker build \
     --build-arg http_proxy=http://<your_proxy_ip>:<your_proxy_port> \ #optional
     --build-arg https_proxy=http://<your_proxy_ip>:<your_proxy_port> \ #optional
+    --build-arg model=pytorch \
+    --build-arg auto_tuning=y \
+    --build-arg hardware=single \
+    --build-arg extra_dep=n \
     -t chronos-nightly:b1 . # You may choose any NAME:TAG you want.
 ```
 According to your network status, this building will cost **15-30 mins**. 
```

Comment:
could we meet this build time?

Reply:
Yeah. According to my test, when all options are specified, it take nearly 27 mins to finish, and when we only install chronos with pytorch, it takes 13-15 mins.